src/net/sf/memoranda/ui/AppFrame.java

```diff
@@ -689,8 +689,7 @@ protected void processWindowEvent(WindowEvent e) {
         }
         else if ((e.getID() == WindowEvent.WINDOW_ICONIFIED)) {
             super.processWindowEvent(new WindowEvent(this,
-                    WindowEvent.WINDOW_CLOSING));
-            doMinimize();
+                    WindowEvent.WINDOW_ICONIFIED));
```

Comment:
I would accept the hypothesis that this is a platform dependent implementation. At any rate, it is not clear why the author was capturing and transforming the event before delegating processing to the super...followed by calling dispose() on the frame.

You're going to facepalm, but I'm leaning towards going back to removing this block. I can't decipher the intent and it causes harm.